src/main/java/bio/terra/common/sam/SamRetry.java

```diff
@@ -53,6 +54,11 @@ protected SamRetry(Duration timeout) {
     R apply() throws ApiException, InterruptedException;
   }
 
+  public static boolean isTimeoutException(ApiException apiException) {
+    return apiException.getCode() == TIMEOUT_STATUS_CODE
+        && apiException.getCause() instanceof SocketTimeoutException;
```

Comment:
worth a quick comment and on what this means to have a timeout exception and why we think it happens today.